src/main/java/com/suse/saltstack/netapi/event/EventStream.java

```diff
@@ -64,13 +72,11 @@ public EventStream() {
      *
      * @param config Contains the necessary details such as EndPoint URL and
      * authentication token required to create the WebSocket.
+     * @throws SaltStackException in case of an error during stream initialization
      */
-    public EventStream(ClientConfig config) {
-        try {
-            initializeStream(config);
-        } catch (SaltStackException e) {
-            e.printStackTrace();
-        }
+    public EventStream(ClientConfig config) throws SaltStackException {
+        maxMessageLength = config.get(ClientConfig.WEBSOCKET_MAX_MESSAGE_LENGTH);
```

Comment:
Going by @isbm 's comment earlier (ie. misunderstanding what this parameter represents), I think that maybe the <= 0 case could be handled here, when config is accessed to initialize maxMessageLength parameter.

    maxMessageLength = (config.get(...) > 0 ? config.get(...) : Integer.MAX_VALUE);

then later we don't have to test for > 0 case and just use it at face value.

This is just a suggestion, it's also OK as is.